custom/theme.scss

```diff
@@ -1,6 +1,6 @@
-$light-background: #e9e5fb;
```

Comment:
Ensure consistent naming conventions for color variables to maintain clarity and consistency. 

Suggested change:
```scss
$light-background: #F8EFF4;
$dark-background: #10091d;
$highlight-text: #F8EFF4;
$light-text: #a7baca;
$lowlight-text: #535d65;
$dark-text: #10091d;
``` 

For reference, see [Sass Guidelines](https://sass-lang.com/styleguide).